pom.xml

```diff
@@ -538,11 +539,11 @@
                 <artifactId>gt-cql</artifactId>
                 <version>${org.geotools.version}</version>
             </dependency>
-            <dependency>
-                <groupId>org.geotools</groupId>
-                <artifactId>gt-jts-wrapper</artifactId>
-                <version>${org.geotools.version}</version>
-            </dependency>
+<!--            <dependency>-->
```

Comment:
It seems that `gt-jts-wrapper` was commented out in a few different pom files, but never removed.